content/docs/guides/neon-authorize-auth0.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Secure your data with Auth0 and Neon Authorize
+subtitle: Implement Row-level Security policies in Postgres using Auth0 and Neon Authorize
+enableTableOfContents: true
+---
+
+<InfoBlock>
+<DocsList title="Sample project" theme="repo">
+  <a href="https://github.com/neondatabase-labs/auth0-nextjs-neon-authorize">Auth0 + Neon Authorize</a>
+</DocsList>
+
+<DocsList title="Related docs" theme="docs">
+  <a href="/docs/guides/neon-authorize-tutorial">Neon Authorize Tutorial</a>
+</DocsList>
+</InfoBlock>
+
+Use Auth0 with Neon Authorize to add secure, database-level authorization to your application. This guide assumes you already have an application using Auth0 for user authentication. It shows you how to integrate Auth0 with Neon Authorize, then provides sample Row-level Security (RLS) policies to help you model your own application schema.
+
+## How it works
+
+Auth0 handles user authentication by generating JSON Web Tokens (JWTs), which are securely passed to Neon Authorize. Neon Authorize validates these tokens and uses the embedded user identity metadata to enforce the [Row-Level Security](https://neon.tech/postgresql/postgresql-administration/postgresql-row-level-security) policies that you define directly in Postgres, securing database queries based on that user identity.
```

Comment:
would be nice to add: "This is all powered by the open source pg_session_jwt Postgres extension" and have a github link there

Reply:
done